### PR TITLE
UI: Update to Ember 3.8 LTS

### PR DESCRIPTION
**Pull request by backspace**
I haven’t yet updated Ember Data, but as of when I’m opening this PR, the tests are passing and the deprecations are fixed, apart from [this existing one](https://github.com/IvyApp/ivy-codemirror/pull/40) that requires a new release of `ivy-codemirror`.

I’ll update Ember Data and Ember Data Model Fragments next 😳

This is currently set to merge into the branch from #5959, but that’s a bit of a side quest that I extracted to its own PR, it can be merged separately, and then I can change this back to merge into `master`.

### CODE REVIEW

File: ui/app/models/job.js [hunk 1]
```diff
@@ -246,7 +244,7 @@ export default Model.extend({
     const id = payload.Name;
 
     this.set('plainId', id);
-    this.set('id', JSON.stringify([id, namespace]));
+    this.set('_idBeforeSaving', JSON.stringify([id, namespace]));
```
Comment (backspace) [inline ui/app/models/job.js]: You [can’t change the id after the record has been created](https://discordapp.com/channels/480462759797063690/486549196837486592/601083103364251659), so this instead saves to a private-y property for later use in the job adapter.

File: ui/app/serializers/node-attributes.js [hunk 2]
```diff
@@ -2,6 +2,6 @@ import ApplicationSerializer from './application';
 
 export default ApplicationSerializer.extend({
   normalize(typeHash, hash) {
-    return this._super(typeHash, { Attributes: hash });
+    return this._super(typeHash, { NodeAttributes: hash });
```
Comment (backspace) [inline ui/app/serializers/node-attributes.js]: `attributes` seems to have become somewhat of a reserved word, so this gives it a redundant-but-functional new name.

**Comment by backspace** [comment]
The changes here are:
* deprecation fixes
* autoformatting that came along with other needed changes
* adaptations for Ember Data and Model Fragments behavioural differences

File: ui/app/serializers/node-attributes.js (hunk 2 above)
Comment (DingoEatingFuzz) [reply ui/app/serializers/node-attributes.js]: Ah, my least favorite part about Ember.

**Comment by backspace** [comment]
👍 I’ll hold off merging until it’s released then, thanks.

**Pull request merged**